core/src/test/java/org/apache/spark/launcher/SparkLauncherSuite.java

```diff
@@ -17,10 +17,13 @@
 
 package org.apache.spark.launcher;
 
+import java.io.File;
 import java.util.Arrays;
 import java.util.HashMap;
 import java.util.Map;
 
+import org.apache.commons.lang.SystemUtils;
+import org.apache.spark.util.Utils;
```

Comment:
Maybe we should reorder imports. (See [databricks/scala-style-guide#imports](https://github.com/databricks/scala-style-guide#imports))